### ISSUE TITLE
Port of fluent validation to python

### ISSUE THREAD

**Issue by SimonGeering** [FluentValidation/FluentValidation#1606]
Hi All, 

Does anyone know if there is a published port of this framework for python, please?

The closes I have been able to find is here, although it does not seem to have been maintained by the original author:
https://github.com/ALPHAvibe/PyValidator

I am a long term user of the c# version and am looking to take advantage of the patterns FluentValidation allows over in a Python project I'm helping with. 

If I were to approach the author of the above with a view to publishing a fork of that python package as "FluentValidation" over on https://pypi.org/ (the python version of nuget.org) would that be something the authors of this c# version would be agreeable to?

- On the one hand, I have no desire to miss-represent what is owned or built by whom, since I would be re-packaging another developers python code and handling the "dev-ops" publication pipeline to get it out into the world with a name to show its heritage as a port of this library. 
- However, on the other hand, I would like to make the capabilities of FluentValidation available to other developers in the python world. As a pro c# dev who is doing a bit of Python on the side as a hobby, I believe they are missing out and could benefit.

In summary, I would like to share the benefits to a wider audience but do so in a way that takes into account everyone involved and ensure all are cool with any intended approach before proceeding further. 

Thanks in advance for taking the time to provide any feedback on this as I know its somewhat out of the ordinary.

**Comment by JeremySkinner** [comment]
Hi, no issues here provided you abide by the terms of the license. If you also want to credit me as the original author in the port's readme I'd also appreciate that.

**Comment by SimonGeering** [comment]
Thanks, Jeremy. 

I have opened an issue with Rob who wrote the port in question
https://github.com/ALPHAvibe/PyValidator/issues/1
and will see where it goes. 

As I noted in my issue over there will obviously be happy to credit yourself as the original designer and author of fluent validation that the port was derived from, alongside the author of the port. 

I'm just the c# dev middle-man trying to get to use his favourite validation library on his dad's python app :-)
